packages/styles/src/modules/pagination.css

```diff
@@ -0,0 +1,191 @@
+/**
+ * Used to split large content into several pages, allowing users to navigate between them instead of displaying all information on a single page.
+ * @name sd-pagination
+ * @status stable
+ * @since 1.0
+ * @variant { simple } sd-pagination--... The pagination format.
+ * @boolean sd-pagination--inverted Inverts the pagination style.
+ */
+
+.sd-pagination {
+  > :not(ul) {
+    position: absolute;
+    overflow: hidden;
+    height: 1px;
+    width: 1px;
+  }
+
+  ul {
+    @apply flex items-center gap-2;
+
+    li a {
+      @apply text-primary flex items-center justify-center;
+
+      &:focus-visible {
+        @apply outline outline-2 outline-offset-2 outline-primary;
+      }
+    }
+
+    /* Previous and Next */
+    li:first-child,
+    li:last-child {
+      @apply w-12 h-12;
+
+      a {
+        @apply w-full h-full;
+      }
+
+      sd-icon {
+        @apply w-6 h-6;
+      }
+    }
+
+    /* Numbers */
+    li {
+      @apply flex items-center justify-center text-center w-8 h-8;
+
+      &:has(a[data-page]) {
+        a {
+          @apply border-b-2 border-b-transparent w-full h-full;
+
+          &[aria-current] {
+            @apply border-b-accent;
+          }
+        }
+      }
+
+      a[aria-current] {
+        @apply text-black;
+      }
+    }
+
+    &:has(> li:nth-child(2) a[aria-current]) > li:nth-child(1) a, /* When current is the first page, apply styles to prev button */
+    &:has(> li:nth-last-child(2) a[aria-current]) > li:nth-last-child(1) a /* When current is the last page, apply styles to next button */ {
+      @apply cursor-not-allowed text-neutral-500;
```

Comment:
Are these the arrows? Can't we just have an has() selector looking for the sd-icons inside?